ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -179,7 +179,7 @@ - (NSString *) getDeviceName {
 
 - (BOOL) isDisplayZoomed {
     #if !TARGET_OS_VISION
-        return [UIScreen mainScreen].scale != [UIScreen mainScreen].nativeScale;
+        return [UIScreen mainScreen].scale < [UIScreen mainScreen].nativeScale;
```

Comment:
@ffraaz, this is an interesting change. 

This implies that `.scale` is _can_ be greater than `.nativeScale` — am I understanding that correctly?

I haven't used this feature, so I don't know.

Reply:
Yes, exactly. On iPhone 12 mini and iPhone 13 mini `scale` is `3.0` and `nativeScale` is `2.88` with default settings.
This leads to the false positives for `isDisplayZoomed` when using `!=` as the comparison operator.
From John Gruber's [iPhone 12 mini review](https://daringfireball.net/2020/11/the_iphone_12_mini_and_iphone_12_pro_max):
> The 12 Mini is also doing something else to pack more content onto the screen at a smaller size: its standard display resolution is ever so slightly scaled down...on the 12 Mini, the display is 2340 x 1080, but standard scale screenshots are 2436 x 1125 — a scaling factor of 96 percent. Software running on the 12 Mini “sees” the display as 2436 x 1125, but everything is downsampled on-the-fly to 2340 x 1080.

Reply:
That is...kind of crazy. Wow! +1 for posting the docs 😆 makes it easy to understand at least